BraveWallet/Extensions/BraveWalletExtensions.swift

```diff
@@ -227,4 +227,9 @@ extension BraveWallet.BlockchainToken {
   var assetBalanceId: String {
     "\(contractAddress)+\(symbol)"
   }
+  
+  var isAuroraSupportedToken: Bool {
+    (contractAddress.isEmpty || WalletConstants.supportedAuroraBridgeTokensContractAddresses.map({ $0.lowercased() })
+      .contains(contractAddress.lowercased())) && chainId == BraveWallet.MainnetChainId
```

Comment:
Not required change: This was a bit tough to read mostly due to length of the contains check and it being split to a second line. If this is the only spot `WalletConstants.supportedAuroraBridgeTokensContractAddresses` should we just lowercase the contract addresses in the constant instead of at runtime? Or we could use case insensitive compare:

```
  var isAuroraSupportedToken: Bool {
    let isSupportedContractAddress = WalletConstants.supportedAuroraBridgeTokensContractAddresses
      .contains(where: { $0.caseInsensitiveCompare(contractAddress.lowercased()) == .orderedSame })
    return (contractAddress.isEmpty || isSupportedContractAddress) && chainId == BraveWallet.MainnetChainId
  }
```